renderer/components/action-bar/controls/main.js

```diff
@@ -85,6 +97,18 @@ MainControls.Left = connect(
 )(Left);
 
 class Right extends React.Component {
+  icon = React.createRef();
+
+  openMenu = () => {
```

Comment:
Should really make a reusable component for this as a Electron dropdown menu is used in many places.